README.md

```diff
@@ -5,12 +5,20 @@ Currently it checks transit.land and transitfeeds.com to verify that feeds are
 listed in an aggregator.
 
 
+## Requirements
+
+* `.env` - Acquire an [api key from transitland][1] and save it to a `.env` file
+  like `TRANSITLAND_API_KEY=SECRET`. Alternatively you can prefix commands with
+  the api key like `TRANSITLAND_API_KEY=SECRET python feed_checker.py [...]`.
+
+* `agencies.yml` - This file can have any structure as the feed checker just
```

Comment:
Can this be changed to note that an agencies.yml file isn't required? An individual URL or a CSV file would work too.